packages/remix-dev/cli/create.ts

```diff
@@ -576,34 +580,36 @@ export async function validateTemplate(input: string) {
       }
       let typeDir = templateType + "s";
       let templateUrl = `https://github.com/remix-run/remix/tree/main/${typeDir}/${name}`;
+      let response;
```

Comment:
I had some trouble coming up with a way to test the failure scenario for in-repo example/template since they're already validated against known-good values.  Any ideas?  Or OK with the current tests?

Reply:
I think we're ok with current tests 👍